src/Simulation/EntryPointDriver/IEntryPoint.cs

```diff
@@ -33,6 +33,14 @@ public interface IEntryPoint
         /// </summary>
         IEnumerable<Option> Options { get; }
 
+        /// <summary>
+        /// Generates payload for Azure Quantum for the entry point.
+        /// </summary>
+        /// <param name="parseResult">The command-line parsing result.</param>
+        /// <param name="settings">The generate Azure payload settings.</param>
+        /// <returns>The exit code.</returns>
+        Task<int> GenerateAzurePayload(ParseResult parseResult, GenerateAzurePayloadSettings settings);
```

Comment:
I'm curious, is it possible to avoid duplicating the param name in the doc comments? E.g. if one renames `parseResult` in the code,  but forgets to rename in the doc comments, then the doc comments will mislead. 

To summarize, is it possible to write like this?
```c#
        /// <summary>
        /// Generates payload for Azure Quantum for the entry point.
        /// </summary>
        /// <returns>The exit code.</returns>
        Task<int> GenerateAzurePayload(
                ParseResult parseResult, /// <param>The command-line parsing result.</param>
                GenerateAzurePayloadSettings settings); /// <param>The generate Azure payload settings.</param>
```

Reply:
When that happens, a warning like the following will appear when building the project:
```
C:\Microsoft\ReposA\qsharp-runtime\src\Simulation\EntryPointDriver\IEntryPoint.cs(59,68): warning CS1573: Parameter 'afhak' has no matching param tag in the XML comment for 'IEntryPoint.Simulate(ParseResult, DriverSettings, string)' (but other parameters do) [C:\Microsoft\ReposA\qsharp-runtime\src\Simulation\EntryPointDriver\Microsoft.Quantum.EntryPointDriver.csproj]
```

Additionally, both IDEs like VS and VSCode underline both the param doc comment and the method parameter itself pointing to the lack of doc comment and doc comment for non-existing param.

Additionally, per the [official documentation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments), documentation comments "must immediately precede a user-defined type (such as a class, delegate, or interface) or a member (such as a field, event, property, or method). "